check-js-types.js

```diff
@@ -39,8 +45,7 @@ const args = [].concat(
     '--checkJs',
     '--noEmit',
     c.flags.strict ? '--strict' : [],
-    '-g',
-    c.input[0]
+    filesList
```

Comment:
This should also solve an issue in case the user wants to check multiple lists of files using globby, for example: `bin/**/*.js lib/**/*.js`